Tests/MeiliSearchUnitTests/SettingsTests.swift

```diff
@@ -42,11 +40,21 @@ class SettingsTests: XCTestCase {
 
     // MARK: Settings
 
+    func testShouldInstantiateFromEmptyData() {
```

Comment:
Where is this function called ?

Reply:
It is a test, test are automatically detected and added by Xcode, no need to call them manually.

But I see your point, for Linux I need to add the test manually at the end of the file, I'm not used to that, I never perform test for Linux.

Reply:
I did not know that. I'm used to adding them at the end of the file mostly also to have a quick way of knowing which tests are done in each file

Reply:
Well, I forgot to add, functions prefixed with `test` will be counted as tests 😄